content/systems/filecoin_token/payment_channels/_index.md

```diff
@@ -2,40 +2,136 @@
 
-Once their transactions have completed, either party can choose to `Close` the channel, the recipient can then `Collect` the `ToPay` amount from the channel. `From` will be refunded the remaining balance in the channel.
+Once their transactions have completed, either party can choose to `Settle` (i.e., close) the channel. There is a 12hr period after `Settle` during which the recipient can submit any outstanding vouchers. Once the vouchers are submitted, the recipient can then `Collect` the `ToPay` amount from the channel. The channel sender (`From` address) will be refunded the remaining balance in the channel (if any).
 
-So we have:
 
-- \[off-chain\] - Two parties agree to a series of transactions (for instance as part of file retrieval) with party **A** paying party **B** up to some **total** sum of Filecoin over time.
-- \[on-chain\] - The [Payment Channel Actor](payment_channel_actor.md) is used, called by A, to open a payment channel `from` A `to` B and a lane is opened to increase the `balance` of the channel, triggering a transaction between A and the payment channel actor.
-At any time, A can open new lanes to increase the total balance available in the channel (e.g. if A and B choose to do more transactions together).
-- \[off-chain\] - Throughout the transaction cycle (e.g. on every piece of data sent via a retrieval deal), party A sends a voucher to party B enabling B to redeem payment from the payment lanes, and incentivizing B to continue providing a service (e.g. sending more data along).
-- \[on-chain\] - At regular intervals, B can `Update` the payment channel balance available `ToSend` with the vouchers received (past their `timeLock`), decreasing the remaining `Value` of the payment channel.
-- \[on-chain\] - At the end of the cycle, past the `MinCloseHeight`, A can choose to `Close` the payment channel.
-- \[on-chain\] - B can choose to `Collect` the amount `ToSend` triggering a payment between the payment channel actor and B.
+## Lanes
+
+In addition, payment channels in Filecoin can be split into `lane`s created as part of updating the channel state with a payment `voucher`. Each lane has an associated `nonce` and amount of tokens it can be `redeemed` for. Lanes can be thought of as transactions for several different services provided by the channel recipient to the channel sender. The `nonce` plays the role of a sequence number of vouchers within a given lane, where a voucher with a higher nonce replaces, but does not invalidate a voucher with a lower nonce - the recipient can still redeem or collect a lower-nonce voucher.
```

Comment:
I believe that submitting a voucher with a higher nonce does actually supersede a voucher with a lower nonce.
So if the client sends (nonce, value) (1,5) then (2,10), the miner can submit (1,5) then (2,10), but cannot submit (2,10) then (1,5).
Note that the lane value = the voucher in the lane with the highest nonce (not the sum of all voucher values)